data/src/main/java/io/gnosis/data/repositories/SafeRepository.kt

```diff
@@ -81,6 +72,27 @@ class SafeRepository(
 
     suspend fun saveSafeMeta(safeMeta: SafeMetaData) = safeDao.saveMeta(safeMeta)
 
+    suspend fun getSafeStatus(safeAddress: Solidity.Address): SafeStatus {
+
+        val safeInfo = transactionServiceApi.getSafeInfo(safeAddress.asEthereumAddressChecksumString())
+
+        val supportedContracts = setOf(
```

Comment:
Just to clarify: We don't support earlier contracts anymore?

What about 1.2.0? Do we need changes on the app to support it? Could we allow this now, so we don't have to release a version when it is used?

Reply:
we will need to add 1.2.0 eth address at some point. Implementation copy values are not incrementally increasing, so we need to list actual values of the supported contracts